corehq/apps/translations/_design/views/popularity/map.js

```diff
@@ -1,10 +0,0 @@
-function (doc) {
-    if ((doc.doc_type === 'Application' && doc.copy_of === null) || doc.doc_type === 'TranslationDoc') {
```

Comment:
`TranslationDoc` was never a doc_type (ever). I've simplified this in the new view to reflect that